.github/CONTRIBUTING.md

```diff
@@ -79,34 +81,35 @@ pnpm i
 
 It may take a while to install dependencies.
 
-### Set up environment variables (optional)
+### Set up database
 
-The root `npm start` is optimized for public release, which carries the `--from-root` parameter. In the dev environment, usually, we read `.env` from the package location instead.
+Create a `.env` file with the following content in the project root, or set the environment variable directly:
 
-- If you already have a `.env` in the project root, move it into `packages/core/` before continuing.
-- If it's a fresh setup, no action is needed now. You can follow the command line questions afterward.
+```env
+DB_URL=postgresql://your-postgres-dsn/logto # Replace with your own
+```
 
-## Start dev
+Then run `pnpm cli db seed` to seed data into your database.
 
-Run the command below in the project root:
+### Database alteration
 
-```bash
-pnpm dev
-```
+If you are upgrading your dev environment from an older version, or facing the `Found undeployed database alterations...` error when starting Logto, you need to deploy the database alteration first.
 
-The command will do several things in order:
+Run `pnpm run alteration deploy` and start Logto again. See [Database alteration](https://docs.logto.io/docs/tutorials/using-cli/database-alteration) for reference.
```

Comment:
```suggestion
Run `pnpm alteration deploy` and start Logto again. See [Database alteration](https://docs.logto.io/docs/tutorials/using-cli/database-alteration) for reference.
```

Reply:
Also this is for developers, so they might just want to run the alteration scripts of "next". So it would be better to let them run `pnpm alteration deploy next` as well

Reply:
added